hit-testing-explainer.md

```diff
@@ -6,6 +6,18 @@ The purpose of this document is to describe a design for enabling developers to
 ## Introduction
 "Hit testing" (aka "raycasting") is the process of finding intersections between 3D geometry and a ray, comprised of an origin and direction. Conceptually, hit testing can be done against virtual 3D geometry or real-world 3D geometry. As WebXR does not have any knowledge of the developer's 3D scene graph, it does not provide APIs for virtual hit testing. It does, however, have information about the real-world and provides a method for developers to hit test against it. Most commonly in WebXR, developers will hit test using `XRInputSource`s or the `XRReferenceSpace` of type `"viewer"` to track where a cursor should be drawn on hand-held devices, or even to bounce a virtual object off real-world geometry. In WebXR, 'inline' and 'immersive-vr' sessions are limited to performing virtual hit tests, while 'immersive-ar' sessions can perform both virtual and real-world hit tests.
 
+## Use-cases & scope
+Main use-cases enabled by hit testing API include:
+
+* Showing a reticle that appears to track the real world surfaces at which the device or controller is pointed.
```

Comment:
I think this is stretching the definition of "reticle" a bit too much. The way I understand it, a reticle is either a 2D concept such as a fixed screen location, or it could be extended to mean a ray going in a specific direction for a controller, but  I think specific 3D positions such as the hit location would be clearer as a separate concept. IMHO the reticle sticks to the phone or controller, not to the target surface.

Reply:
s/"a reticle"/"an object"? After all, apps just want to be able to know where to place _something_. Let me think of something that avoids using the terms or describing the processes that'll be defined later. If I cannot come up with anything satisfying I'll fall back to using more specific terms.

Reply:
Rephrased slightly to avoid using "reticle", let me know what you think.

Reply:
Looks good, thanks!